plugin/build.gradle

```diff
@@ -63,6 +63,7 @@ dependencies {
     implementation group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
     implementation group: 'org.apache.commons', name: 'commons-text', version: '1.10.0'
     implementation "org.apache.logging.log4j:log4j-slf4j-impl:2.19.0"
+    implementation group: 'org.json', name: 'json', version: '20231013'
```

Comment:
Lets use jackson databind which is already packaged currently[1] with ml-commons ? 
We try to avoid using multiple JSON creators and parsers in the core. 

[1] https://github.com/opensearch-project/ml-commons/blob/main/plugin/src/main/java/org/opensearch/ml/utils/RestActionUtils.java#L50

Reply:
Makes sense. Just removed this dependency and used ObjectMapper instead of JsonObject.